### PR TITLE
services/horizon: Horizon 2.9.0 CHANGELOG

### PR DESCRIPTION
**Pull request by bartekn**

### CODE REVIEW

**Review by tamirms — APPROVED** [review]

File: services/horizon/CHANGELOG.md [hunk 1]
```diff
@@ -3,17 +3,35 @@
 All notable changes to this project will be documented in this
 file. This project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## v2.9.0
 
-* Update `/paths` endpoint to take liquidity pools into account when searching for possible routes between assets ([3921](https://github.com/stellar/go/pull/3921)).
+**Upgrading to this version from <= v2.8.3 will trigger a state rebuild. During this process (which will take at least 10 minutes), Horizon will not ingest new ledgers.**
+
+This release comes with support for Protocol 18 ([CAP 38](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0038.md): Automated Market Makers).
```
Comment (ire-and-curses) [inline services/horizon/CHANGELOG.md:10]:
```suggestion
**Protocol 18 support:** This release adds support for Protocol 18 ([CAP 38](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0038.md): Automated Market Makers).
```

File: services/horizon/CHANGELOG.md [hunk 2]
```diff
@@ -3,17 +3,35 @@
...
+
+### DB Schema Migration
+
+* This release comes with a DB migration removing `offer_id` field from `history_trades` table and adding new tables related to AMM. It should not take more than 15 minutes.
```
Comment (ire-and-curses) [inline services/horizon/CHANGELOG.md:14]:
```suggestion
* This release comes with a DB migration removing `offer_id` field from `history_trades` table and adding new tables related to AMM. It should not take more than 15 minutes to complete the migration.
```

File: services/horizon/CHANGELOG.md [hunk 3]
```diff
@@ -3,17 +3,35 @@
...
+
+### Breaking changes
 
-### Breaking
 * The `--ingest` flag is set by default. If `--captive-core-config-path` is not set, the config file is generated based on network passhprase ([3783](https://github.com/stellar/go/pull/3783)).
 
+### Changes
+
+* **[CAP 38](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0038.md): Automated Market Makers) support.**
```
[the remark this answers is not in the document]
Comment (ire-and-curses) [reply services/horizon/CHANGELOG.md:22]: We should summarise them and then point to the docs I think. People are going to be curious about the new functionality and we should have a way for them to learn more.

**Review by ire-and-curses — APPROVED** [review]

File: services/horizon/CHANGELOG.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (bartekn) [reply services/horizon/CHANGELOG.md:22]: OK, will create a separate PR fixing this post RC release.

**Pull request merged**